examples/onnxruntime/optimization/token-classification/requirements.txt

```diff
@@ -0,0 +1,5 @@
+seqeval
+datasets >= 1.8.0
```

Comment:
For `conll2003`, `datasets >= 1.18.0` is demanded.

Reply:
Good catch, thanks !